HW03/andrei_godes/spec/andrei_godes_hw_03_t_01_rspec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative '../andrei_godes_hw_03_t_01'
+require_relative 'spec_helper'
+
+RSpec.describe Homework3 do
+  subject { described_class.new }
```

Comment:
you can do it this way and use `is_expected` in all tests 
```suggestion
  subject { described_class.new(log) }
```